README.md

```diff
@@ -1033,6 +1033,12 @@ sudo systemctl enable --now pcscd.service
 
 **macOS**
 
+```console
+brew install gnupg
```

Comment:
the equivalent of `pscs-tools` for Homebrew is `pcsc-lite`, but I'm not sure we need it. The rest of the commands work even when I don't install it.